extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -137,20 +175,54 @@ export class AmpVizVega extends AMP.BaseElement {
    */
   renderGraph_() {
     return new Promise((resolve, reject) => {
-      vega.parse.spec(this.data_, (error, chart) => {
+      this.vega_.parse.spec(this.data_, (error, chartFactory) => {
```

Comment:
`#data` is no longer guaranteed since `#renderGraph` may be called from `#onLayoutMeasure`. Is that intentional?